integration_tests/tests/features/WalletTransfer.feature

```diff
@@ -90,3 +90,21 @@ Feature: Wallet Transfer
     When I claim an HTLC refund transaction with wallet WALLET_A at fee 20
     When mining node MINER_2 mines 6 blocks
     Then I wait for wallet WALLET_A to have at least 9000000000 uT
+
+  Scenario: As a wallet I want to claim a Blake2 HTLC refund transaction
+    Given I have a seed node NODE
+    # Add a 2nd node otherwise initial sync will not succeed
+    When I have 1 base nodes connected to all seed nodes
+    When I have wallet WALLET_A connected to all seed nodes
+    When I have wallet WALLET_B connected to all seed nodes
+    When I have wallet WALLET_C connected to all seed nodes
+    When I have mining node MINER connected to base node NODE and wallet WALLET_A
+    When I have mining node MINER_2 connected to base node NODE and wallet WALLET_C
+    When mining node MINER mines 10 blocks
+    Then I wait for wallet WALLET_A to have at least 10000000000 uT
+    When I broadcast a blake2 HTLC transaction with 5000000000 uT from wallet WALLET_A to wallet WALLET_B with timelock 12 at fee 20
+    When mining node MINER_2 mines 16 blocks
+    When I wait 5 seconds
+    When I claim an HTLC refund transaction with wallet WALLET_A at fee 20
+    When mining node MINER_2 mines 6 blocks
+    Then I wait for wallet WALLET_A to have at least 9000000000 uT
```

Comment:
The numbers here seem off. I know the _Scenario: As a wallet I want to claim a HTLC refund transaction_ looks exactly the same, but that one is also wrong (please fix it as well).

If we mine 10 blocks we will have way more than 10 000 000 000 uT, so to test eventually after the refund has been claimed if we have at least 9 000 000 000 uT is a non-test. The 1st balance enquiry tells us we have enough funds for the _blake2 HTLC transaction_. After that transaction we should actually know the correct balance, or close to it, for a valid assert at the end. My suggestion:
- _When mining node MINER mines 10 blocks_
- **_Then I wait for wallet WALLET_A to have at least `BALANCE_A` uT_** (`BALANCE_A` must be a few Tari less than what is available to allow for fees and some change)
- _When I broadcast a blake2 HTLC transaction with `BALANCE_A` uT from wallet WALLET_A to wallet WALLET_B with timelock 12 at fee 20_
- _When mining node MINER_2 mines 16 blocks_
- **_I wait for wallet WALLET_A to have less than `BALANCE_A` uT_**
- ~_ When I wait 5 seconds_~
- _When I claim an HTLC refund transaction with wallet WALLET_A at fee 20_
- _When mining node MINER_2 mines 6 blocks_
- **_Then I wait for wallet WALLET_A to have at least `BALANCE_A` uT_**